hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/BlockStreamConfig.java

```diff
@@ -38,4 +42,6 @@ public record BlockStreamConfig(
         @ConfigProperty(defaultValue = "true") @NetworkProperty boolean compressFilesOnCreation,
         @ConfigProperty(defaultValue = "32") @NetworkProperty int serializationBatchSize,
         @ConfigProperty(defaultValue = "32") @NetworkProperty int hashCombineBatchSize,
-        @ConfigProperty(defaultValue = "1") @NetworkProperty int roundsPerBlock) {}
+        @ConfigProperty(defaultValue = "1") @NetworkProperty int roundsPerBlock,
+        @ConfigProperty(defaultValue = "localhost") String address,
```

Comment:
Nit: we may want to have a more descriptive name for this property, or some further documentation, specifically to address: wow does this differ from the already-configured address and port for the node?

Reply:
Addressed, tyvm!